conans/test/functional/configuration/profile_test.py

```diff
@@ -743,16 +743,20 @@ def configure(self):
     """
     conanfile += configure
     client.save({"conanfile.py": conanfile})
-    client.run("create . -s foo:build_type=Debug")
+    client.run("create . -s foo*:build_type=Debug")
     assert "I'm foo and my build type is Debug" in client.out
 
-    client.run("create . -s foo:build_type=Debug -s &:build_type=Release")
+    client.run("create . -s foo*:build_type=Debug -s &:build_type=Release")
     assert "I'm foo and my build type is Release" in client.out
 
-    # The order doesn't matter
-    client.run("create . -s &:build_type=Release -s foo:build_type=Debug ")
+    # The order DOES matter !!! This behavior changed!
+    client.run("create . -s foo*:build_type=Debug -s &:build_type=Release")
```

Comment:
Isn't this identical to line 749?
Also this is one of the lines that is difficult to understand. Why using ``&`` if we have the package name? For convenience?

Reply:
| Why using & if we have the package name? For convenience?

hahaha, yes, you use `&` because you can. In this case you could use the name but the "consumer" mechanism, unless we modify it, allows to use `&` in a create like before.

Yes, the line is repeated. I'll remove it.